From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Albert I <kras@raphielgang.org>
Date: Tue, 23 Aug 2022 06:16:52 +0000
Subject: [PATCH 182/226] PackageManagerServiceUtils: Wipe package cache on
 upgrade even on user build

As we're spoofing builds by pretending it to be user, let's just do this
for everyone.

Change-Id: I5d967c25bd9c4890549d5d20f5e66b0c6845528c
---
 .../java/com/android/server/pm/PackageManagerServiceUtils.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/pm/PackageManagerServiceUtils.java b/services/core/java/com/android/server/pm/PackageManagerServiceUtils.java
index 121dc089d6..b77b8b1eda 100644
--- a/services/core/java/com/android/server/pm/PackageManagerServiceUtils.java
+++ b/services/core/java/com/android/server/pm/PackageManagerServiceUtils.java
@@ -1331,7 +1331,7 @@ public class PackageManagerServiceUtils {
         // NOTE: When no BUILD_NUMBER is set by the build system, it defaults to a build
         // that starts with "eng." to signify that this is an engineering build and not
         // destined for release.
-        if (isUserDebugBuild && incrementalVersion.startsWith("eng.")) {
+        if (incrementalVersion.startsWith("eng.")) {
             // Heuristic: If the /system directory has been modified recently due to an "adb sync"
             // or a regular make, then blow away the cache. Note that mtimes are *NOT* reliable
             // in general and should not be used for production changes. In this specific case,
